tests/consumers/test_schemas.py

```diff
@@ -42,6 +42,8 @@ def _generate_tests() -> Iterator[Case]:
 
         try:
             for example in sentry_kafka_schemas.iter_examples(topic.value):
+                if "replay" in str(example.path):
+                    continue
```

Comment:
@cmanallen this also needs to be reverted once replays is fixed

Reply:
Its reverted in this PR https://github.com/getsentry/snuba/pull/5342

Reply:
Oh wait no its not... I swear I removed that line of code.  Must have been a change to the master branch.  I will remove it when the time comes 👍